### DISCUSSION TITLE
Pressure development issue

### DISCUSSION THREAD

**Discussion by Ranhill** [firemodels/fds#13533] [Q&A] [answered]
Hi, 

I have a mechanically ventilated compartment where there is one air inlet and one air outlet. A fire test has been conducted in the compartment where HRR resulting from fire on wooden trays stacked over one another at 0.2m (4 trays) was recorded along with the change in the pressure in the room (room was very tight in sealing), and volumetric flow rates in the ducts. I used pre-recorded volumetric flow rates for ducts in the simulation and fire was having hrr ramped as per test result. The fire hrr output in the simulation and volumetric flow rates in the simulation matched closely to the test. But there was huge difference in the pressure. 

I have kept the geometry and leakage close to the real setup. Only the difference is the fire in the simulation where I just have one slab area like a single tray which is having HRRPUA with ramped HRR- F values for whole fire. I have not created multiple trays in the simulation. 

Can anyone please guide where could be the problem? I am using FDS version 6.8.0.

![v vs p](https://github.com/user-attachments/assets/497e834b-dbef-48c1-a8f2-5e611c3b4d6b)

**Comment by drjfloyd** [comment]
You have removed numbers from the axes of your plot so no idea what this error actually represents. Keep in mind all your experimental measurements have uncertainty, some combination of:

-the measured pressure is biased low
-the FDS fire size being biased high and or the measured fire size being biased low
-the measured leakage area winding up smaller than the real value
-the exhaust flow rate being biased slightly low
-the supply flow rate being biased slightly high

would result in FDS predicting higher pressures.

**Reply by Ranhill** [reply]
Hi, thanks for the answer. I removed the numbers because of confidentiality. The presure difference is huge. As for example pressure in test is 200 Pa but in simulation in 6000 Pa. How come there be such a huge difference given that hrr and volumetric flow rates are correct? Uncertainity has been already accounted and they are low.

Another strange thing I have found. I somehow extracted the quadratic fan parameters for inlet and outlet fan and used it in the simulation. Although results are not very good but they are close of real pressure with average difference of 100 Pa.

**Reply by drjfloyd** [reply]
I guarantee you that no measurment in your experiment has 100 % correct values. Every measurement technology has some error associated with it. When you fix all supply and exhaust, then you can magnify the impact of measurement errors. Modeling with a quadratic model allows the model to responds as a real world ventilation system would would by dynamically changing the flow to match the current pressure. It doesn't necessarily  take a lot of error in HRR or flow to result in large pressure differences over time.

**Reply by Ranhill** [reply]
Thanks! I agree that no measurement in any experiment is 100% correct. I will try to check as per your suggestions.

**Comment by mcgratta** [comment]
There is something else to consider -- the low Mach assumption in FDS. We see its effects mostly in tunnels, but it occurs in other configurations as well. Long story short, changes in pressure that are caused by fluctuations in the HRR or changes in ventilation rates lead to short-lived spikes in pressure. The reason is that small pressure fluctuations that bounce around the actual test enclosure are "filtered" out of the equation set, and replaced by larger pressure oscillations that only make sense as a time-average. Those orange spikes in your plot are ramifications of the low Mach assumption. They are not physically correct except as a time-average. This is my assumption, having not seen the actual case.

**Reply by Ranhill** [reply]
Thanks for the input. I will try to see what can I do.

**Reply by mcgratta** [reply, marked as answer]
One thing you can do is to add a line like this if you have not already:
```
&PRES MAX_PRESSURE_ITERATIONS=50 /
```
This matters only if you are using multiple meshes. It forces FDS to iterate the pressure solution more than its default to achieve a more accurate solution. But even with this, you still will see the pressure oscillations.

**Reply by Ranhill** [reply]
Ok, thanks! I will use that pressure iteration max.
